yarn-project/sequencer-client/src/sequencer/sequencer.ts

```diff
@@ -430,7 +430,7 @@ export class Sequencer {
     const bufferSeconds = maxAllowedTime - secondsIntoSlot;
 
     if (bufferSeconds < 0) {
-      this.log.warn(`Too far into slot to transition to ${proposedState}`, { maxAllowedTime, secondsIntoSlot });
+      this.log.debug(`Too far into slot to transition to ${proposedState}`, { maxAllowedTime, secondsIntoSlot });
```

Comment:
This was getting logged twice: once here, once where we logged the thrown `SequencerTooSlowError`.